pkg/image/factory.go

```diff
@@ -223,6 +242,19 @@ func (f *Factory) getCacheSize() (*resource.Quantity, error) {
 	return &c, nil
 }
 
+func (f *Factory) makeBuildHistoryLimit(buildHistoryLimit string) (*int64, error) {
+	if buildHistoryLimit == "" {
+		return nil, nil
+	}
+
+	value, err := strconv.ParseInt(buildHistoryLimit, 10, 64)
+	if err != nil || value < 1 {
+		return nil, errors.New("must provide a valid build history limit >1")
+	}
+
+	return &value, nil
+}
+
```

Comment:
This func isn't being used

Reply:
updated that this function is being used